community/values/src/test/java/org/neo4j/values/storable/ValueComparisonTest.java

```diff
@@ -123,7 +123,8 @@
             datetime(2018, 2, 1, 22, 30, 0, 0, "-02:00"),
             datetime(2018, 2, 2, 1, 30, 0, 0, "+01:00"), // offsets by offsetSecond
             datetime(2018, 2, 2, 1, 30, 0, 0, "Europe/Berlin"), // same offset as +01:00, but name zones come after offsets ...
-            datetime(2018, 2, 2, 1, 30, 0, 0, "Europe/Stockholm"), // ... alphabetically
+            datetime(2018, 2, 2, 1, 30, 0, 0, "Europe/Prague"), // ... alphabetically
+            datetime(2018, 2, 2, 1, 30, 0, 0, "Europe/Stockholm"),
```

Comment:
I'm missing a test case that orders `"Europe/London"` before `+0100` (or something similar to that).